fre/frepp/configureScriptYAML.py

```diff
@@ -23,66 +24,46 @@ def validateYaml(file):
 ###################
 @click.command()
 
-def yamlInfo(y,experiment,platform,target):
+def yamlInfo(yamlfile,experiment,platform,target):
   e = experiment
   p = platform
-  t = target 
+  t = target
+  yml = yamlfile
 
-  with open(y,'r') as f:
+  with open(yml,'r') as f:
     y=yaml.safe_load(f)
-    yml = validateYaml(y)
+    valyml = validateYaml(y)
```

Comment:
`y` is no longer set.. should this be `validateYaml(yml)` instead?

Reply:
The yamlfile is being saved as `yml`, which is being opened as `f` and then `safe_loaded` as `y`. So the loaded yaml is still `y`.

Reply:
Before I had overwritten variables accidentally, using `y` multiple times I think

Reply:
Gotcha!